test/priors/test_multivariate_normal_prior.py

```diff
@@ -16,12 +16,15 @@ def test_multivariate_normal_prior_to_gpu(self):
             self.assertEqual(prior.scale_tril.device.type, "cuda")
             self.assertEqual(prior.precision_matrix.device.type, "cuda")
 
-    def test_multivariate_normal_prior_validate_args(self):
-        # This should be a ValueError in pytorch, see pytorch Github issue #11997
-        with self.assertRaises(ValueError):
-            mean = torch.tensor([0.0, 1.0])
-            cov = torch.tensor([[1.0, 2.0], [2.0, 0.5]])
-            MultivariateNormalPrior(mean, covariance_matrix=cov, validate_args=True)
+    # def test_multivariate_normal_prior_validate_args(self):
```

Comment:
Hmm I do correctly get a ValueError here. Maybe it was a bad short-lived commit upstream? Let.s add this back in (and remove the comment)